OpenChange/MAPIStoreSOGo.m

```diff
@@ -144,7 +145,7 @@
   GSRegisterCurrentThread();
 
   if (initialization_done) {
-      DEBUG(0, ("SOGo backend already initialized.\n"));
+      OC_DEBUG(5, "SOGo backend already initialized.\n");
```

Comment:
As you are changing all log messages on this file, I'd add `[SOGo]` tag here so all messages will have same format :angel: